bench-exchange/src/bench.rs

```diff
@@ -991,7 +1012,13 @@ pub fn airdrop_lamports(client: &Client, drone_addr: &SocketAddr, id: &Keypair,
             Ok(transaction) => {
                 let signature = client.async_send_transaction(transaction).unwrap();
 
-                if let Ok(Some(_)) = client.get_signature_status(&signature) {
+                for _ in 0..30 {
```

Comment:
Probably should be.

Reply:
For these I'm not sure it is the right place to do it since for instance the BankClient version will probably not need these, although I guess it succeeds on the first iteration and then doesn't do the wait so maybe it's ok.